src/main/java/com/animalshelter/animalshelterapp/entity/User.java

```diff
@@ -0,0 +1,70 @@
+package com.animalshelter.animalshelterapp.entity;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.util.Objects;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "game")
+    private String game;
+    @Column(name = "phoneNumber")
+    private BigInteger phoneNumber;
```

Comment:
Поменяй название колонки и тип поля:
```suggestion
    @Column(name = "phonenumber")
    private Long phoneNumber;
```